webui/react/src/pages/Admin/UserManagement.tsx

```diff
@@ -483,32 +485,35 @@ const UserManagement: React.FC<Props> = ({ onUserCreate }: Props) => {
     <>
       <Section className={css.usersTable}>
         <div className={css.actionBar} data-testid="actionRow">
-          <Row>
+          <Row wrap>
             <Column>
-              <Row>
+              <Row width="fill" wrap>
                 {/* input is uncontrolled */}
                 <Input
                   allowClear
                   data-testid="search"
                   defaultValue={nameFilter}
                   placeholder="Find user"
                   prefix={<Icon color="cancel" decorative name="search" size="tiny" />}
+                  width={isMobile ? '100%' : 'max-content'}
                   onChange={handleNameSearchApply}
                 />
                 <Select
                   data-testid="roleSelect"
                   options={roleOptions}
+                  placeholder="Role"
                   searchable={false}
                   value={roleFilter}
-                  width={120}
+                  width={isMobile ? 'calc(100vw - padding)' : 120}
```

Comment:
Please ensure this code is actually running in mobile -- i'm not seeing this style (or the other calc style) applied when in mobile.